lte/gateway/c/oai/tasks/nas5g/include/M5GCommonDefs.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
```

Comment:
many of the definitions here look the same as the definitions in /Users/kozat/magma/lte/gateway/c/oai/common/common_defs.h? Except for the logging related parts, is there a reason to redefine them here again?

Reply:
redefined definitions are removed.